README.md

```diff
@@ -11,7 +11,7 @@ Add this package as requirement to your django app requirements.txt
 
 ```
 # requirements.txt
-log_storage==0.2
+log_storage==0.4.0
```

Comment:
Perhaps we should remove the version here so we don't have to update it every time we bump the version?